keps/sig-windows/2578-windows-conformance/README.md

```diff
@@ -0,0 +1,474 @@
+- The behavior of the `RunAsUserName` field for Windows pods should be that it is supported (i.e. pods can use this), but that there are no guarantees around volume permissions and access when using this field.
+
+#### Network Policies
+
+Note that we do not include UDP, ipv6, in the initial definitions for NetworkPolicy conformance on Windows. Since NetworkPolicy testing support on Windows clusters was just recently added to Kubernetes core, we err on the side of a conservative definition for Windows NetworkPolicy which is sufficient to address the needs of *most* legacy network security applications.
+
+- Ability to protect IPv4 pods from accessing other pods when TCP NetworkPolicies are present that block specific pod connectivity.
+- Ability to protect IPv4 pods from accessing other pods when TCP NetworkPolicies are present that block specific namespace connectability.
+
+#### Windows Advanced Networking and Service Proxying
+
+These apply to Windows Server 2019 and up (we expect windows 2004 to be out of support once this KEP merges).
+
+- Ability to support IPv6 and IPv4 services for accessing internal Linux pods from Windows pods over ClusterIP Endpoints.
+- Ability to support IPv6 and IPv4 services for accessing internal Linux pods from Windows pods over NodePort Endpoints.
+- Ability to support IPv6 and IPv4 services for accessing internal Windows pods from Linux pods over ClusterIP Endpoints.
+- Ability to support IPv6 and IPv4 services for accessing internal Windows pods from Linux pods over NodePort Endpoints.
+- Ability to run IPv4/IPv6 dual-stack networking (on supported OS Versions).
+
+Note that, ipv4 and ipv6 are both known to be highly CNI (overlay vs non-overlay) and OS (2019 vs 2022) dependent.
+
+#### Windows Worker Configuration
+
+Workers node settings that must be compliant.
+
+- Ability to assess node's Microsoft Defender exclusion set for required processes (containerd).
+
+### Implementation
+
+This section details the implementation of a "tool" which verifies these clusters must allow for the following, differentiated set of verifications, specified above.
+
+- Core Conformance
+- HostProcess
+- ActiveDirectory
+- NetworkPolicy
+- ServiceProxy
+
+As an example, if a Windows verification run for 1.22 was occurring, the highest semver close to 1.22, could be used
+for running NetworkPolicy verification.  In this case, 1.21.  This would allow for backward compatibility of this file
+so that if a ginkgo tag changed, a new "test" with a new "version" and the SAME operationalReadinessDescription could be added.
+
+As part of this KEP, we will finish the ongoing ginkgo tag curation at https://docs.google.com/spreadsheets/d/1Pz7-AUZ9uxMBwFx7ZC2U6dBfYpMUdOUi1UivBRVDw7I/edit#gid=0 - to live in the a `sig-windows` repository as an reference summarization and a convinient grouping of the tests.
+This KEP implements a layer of abstraction mainly in a YAML/JSON, which defines each item described below containing
```

Comment:
how is this yaml going to be consumed by sonobouy or e2e.test?  How is maintained? this looks fairly manual to list every test

Reply:
Yup, its definetly manual.  
No plans for long term maintainance.  Its a one time task to define the operational readiness tags for 1.23/24 really.  After that, like anything else, the update lifecycle process for it will depend on how well it serves the hard requirements of the KEP's actual deliverables.

Reply:
The prototype answered many of my questions here.  We do need to think about how we version these json files per release but we can do that for moving to beta